fbw-a380x/src/wasm/systems/a380_systems/src/hydraulic/mod.rs

```diff
@@ -1753,6 +1769,11 @@ impl A380Hydraulic {
             Ratio::new::<ratio>(0.01),
         );
 
+        let left_flaps_factory = A380FlapsFactory::a380_flaps_factory(context, "LEFT");
+        let right_flaps_factory = A380FlapsFactory::a380_flaps_factory(context, "RIGHT");
+        let left_slats_factory = A380SlatsFactory::a380_slats_factory(context, "LEFT");
+        let right_slats_factory = A380SlatsFactory::a380_slats_factory(context, "RIGHT");
```

Comment:
The name of the variables is bit misleading here since they actually don't "contain" the factory but the already constructed object.